python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -31,14 +34,15 @@ def assert_arg_type_is_supported(ty):
     "mri1": np.bool_,
     "mri8": np.int8,
     "mri32": np.int32,
-    "mri64": np.int64
+    "mri64": np.int64,
+    "mrf16": np.float16
 }
 elemental_type_to_ctype = {
     "i1": ctypes.c_bool,
     "i8": ctypes.c_byte,
     "i64": ctypes.c_int,
     "f32": ctypes.c_float,
-    "f64": ctypes.c_double
+    "f64": ctypes.c_double,
```

Comment:
I think there should be an f16 case here

Reply:
This only adds support for the f16's inside a memref and not as an elemental type.